front/app/modules/commercial/content_builder/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { ModuleConfiguration } from 'utils/moduleUtils';
+
+const configuration: ModuleConfiguration = {
+  routes: {
+    admin: [
+      {
+        path: '/:locale/admin/projects/:projectId/content-builder',
+        name: 'content-builder',
+        container: () => import('./admin/containers/index'),
+      },
+    ],
+  },
+  outlets: {
+    'app.containers.Admin.projects.all.container.edit.description.contentBuilder':
+      (props) => {
+        const route = `/admin/projects/${props.projectId}/content-builder`;
```

Comment:
Let's extract this into a component where we get the projectId from the router. For that, you need the `withRouter` HOC. Adding it will allow you access to the params prop which will allow you to do:
``` const { projectId } = params;```